packages/theming/apple-theme/src/fluentAppleColors.ts

```diff
@@ -0,0 +1,316 @@
+import { DynamicColorMacOS } from 'react-native-macos';
+
+/** Palette of colors defined in FluentUI Apple */
+export interface FluentUIAppleBasePalette {
+  blue10: ColorValue;
+  blueMagenta20: ColorValue;
+  blueMagenta30: ColorValue;
+  communicationBlue: ColorValue;
+  communicationBlueShade10: ColorValue;
+  communicationBlueShade20: ColorValue;
+  communicationBlueShade30: ColorValue;
+  communicationBlueTint10: ColorValue;
+  communicationBlueTint20: ColorValue;
+  communicationBlueTint30: ColorValue;
+  communicationBlueTint40: ColorValue;
+  cyan20: ColorValue;
+  cyan30: ColorValue;
+  cyanBlue10: ColorValue;
+  cyanBlue20: ColorValue;
+  dangerPrimary: ColorValue;
+  dangerShade10: ColorValue;
+  dangerShade20: ColorValue;
+  dangerShade30: ColorValue;
+  dangerTint10: ColorValue;
+  dangerTint20: ColorValue;
+  dangerTint30: ColorValue;
+  dangerTint40: ColorValue;
+  gray20: ColorValue;
+  gray25: ColorValue;
+  gray30: ColorValue;
+  gray40: ColorValue;
+  gray50: ColorValue;
+  gray100: ColorValue;
+  gray200: ColorValue;
+  gray300: ColorValue;
+  gray400: ColorValue;
+  gray500: ColorValue;
+  gray600: ColorValue;
+  gray700: ColorValue;
+  gray800: ColorValue;
+  gray900: ColorValue;
+  gray950: ColorValue;
+  green10: ColorValue;
+  green20: ColorValue;
+  magenta10: ColorValue;
+  magenta20: ColorValue;
+  magentaPink10: ColorValue;
+  orange20: ColorValue;
+  orange30: ColorValue;
+  orangeYellow20: ColorValue;
+  pinkRed10: ColorValue;
+  presenceAvailable: ColorValue;
+  presenceAway: ColorValue;
+  presenceBlocked: ColorValue;
+  presenceBusy: ColorValue;
+  presenceDnd: ColorValue;
+  presenceOffline: ColorValue;
+  presenceOof: ColorValue;
+  presenceUnknown: ColorValue;
+  red10: ColorValue;
+  red20: ColorValue;
+  successPrimary: ColorValue;
+  successShade10: ColorValue;
+  successShade20: ColorValue;
+  successShade30: ColorValue;
+  successTint10: ColorValue;
+  successTint20: ColorValue;
+  successTint30: ColorValue;
+  successTint40: ColorValue;
+  warningPrimary: ColorValue;
+  warningShade10: ColorValue;
+  warningShade20: ColorValue;
+  warningShade30: ColorValue;
+  warningTint10: ColorValue;
+  warningTint20: ColorValue;
+  warningTint30: ColorValue;
+  warningTint40: ColorValue;
+}
+
+/** A set of control tokens used on the FluentUI Apple macOS button */
+export interface FluentAppleButtonTokens {
+  brandBackgroundDisabled: ColorValue;
+  brandForegroundDisabled: ColorValue;
+  neutralBackground2: ColorValue;
+  neutralBackground3: ColorValue;
+  neutralForeground2: ColorValue;
+  neutralForeground3: ColorValue;
+  neutralInverted: ColorValue;
+  neutralStroke2: ColorValue;
+}
+
+export type FluentUIApplePalette = FluentUIAppleBasePalette & FluentAppleButtonTokens;
+
+export function getFluentUIApplePalette(): FluentUIApplePalette {
+  return {
+    blue10: '#4F6BED',
+    blueMagenta20: '#8764B8',
+    blueMagenta30: '#5C2E91',
+    communicationBlue: DynamicColorMacOS({
```

Comment:
How does DynamicColorMacOS work, out of curiosity?  Is this like PlatformColor where it's handled natively or is this using the appearance module (I think it's called) to do this dynamically from the JS-side?

Reply:
I'm all for doing what's most ergonomic, but it might also be good to lean into shared patterns where appropriate to make it easier to interpret.

Reply:
I think DynamicColorMacOS is simply analogous to [DynamicColorIOS](https://reactnative.dev/blog/2020/07/06/version-0.63#native-colors-platformcolor-dynamiccolorios) which was introduced with 0.63. I think it handles everything natively and supersedes the appearance API. I'm not opposed to just having a light/dark palette that the theme switches as that may be easier. However, I did notice PlatformColor() supersedes the react-native appearance API and will just follow whatever the system has